src/components/Pricing/HeroPricingWelcome.tsx

```
@@ -117,212 +133,478 @@ const HeroPricingWelcome: React.FC<HeroProps> = ({
+											<span className='text-center text-[9px] text-[#919B9F] uppercase font-semibold tracking-[1px] leading-[12px]'>
+                        Select Frequency
+											</span>
+											<div className='relative overflow-hidden w-full rounded-[100px] h-[40px] px-1 bg-grey-50'>
+												<div className='relative grid h-full grid-cols-2'>
+													<button
+														onClick={ () => setLiteFrequency('semi annual') }
+														className={ clsxm(
+															'text-xs !leading-normal h-full flex items-center justify-center cursor-pointer whitespace-nowrap relative',
+															liteFrequency === 'semi annual' ? 'text-white z-10' : 'text-grey-400'
+														) }
+													>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)